ports/unix/Makefile

```
@@ -1,3 +1,20 @@
+# Fail fast when pkg-config is not installed and required
+PKG_CONFIG_REQUIRED = 0
+
+ifeq ($(MICROPY_BLUETOOTH_BTSTACK),1)
+PKG_CONFIG_REQUIRED = 1
+endif
+
+ifeq ($(MICROPY_STANDALONE),1)
```

`pkg-config` is used in the `else` of this check so it needs to be negated.

```suggestion
ifneq ($(MICROPY_STANDALONE),1)
```